eks/cluster-loader/configs/cluster-autoscaler/hollowdeployment.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
```

Comment:
This file doesn't look like its used anywhere.

Reply:
The point was to have the necessary changes to make it kubemark possible if one would like to switch back.

Reply:
Ah -- this is what git history is for. I think we should leave the old CA as is. It's compatible with both hollow and real nodes.

Reply:
Oh -- wait, the affinity is required. You could change
requiredDuringSchedulingIgnoredDuringExecution
preferredDuringSchedulingIgnoredDuringExecution

which would keep compatibility with both approaches.

The key is that you dont want to add a new feature while breaking an old one.